src/app/store.ts

```diff
@@ -1,10 +1,16 @@
 import { configureStore, ThunkAction, Action } from '@reduxjs/toolkit';
 // eslint-disable-next-line import/no-cycle
 import counterReducer from '../features/counter/counterSlice';
+import usersSlice from '../features/usersSlice';
+import postsSlice from '../features/postsSlice';
+import commentsSlice from '../features/commentsSlice';
 
 export const store = configureStore({
   reducer: {
     counter: counterReducer,
```

Comment:
The 'counter' reducer is not needed according to the task requirement. You might want to remove this line if the counter feature is not used anymore.